### PR TITLE
Linking JS and HTML code

### PR DESCRIPTION
**Pull request by oOoOoOo0o**
You can test the frontend.html to see if it works

### CODE REVIEW

**Review by ShaveYourEyebrows — APPROVED** [review]
Looking good, but is it super necessary to have an entire separate directory just for the PNG?

File: OBlock/src/enrolment.js [hunk 1]
```diff
@@ -1,44 +1,66 @@
...
             }
         }
         if (!found) {
+            alert("Student does has not completed all of the requisite modules!");
             return false;
         }
     }
 
+    alert("Enrolment successful!");
+    module.enrolledStudents.push(student);
```
Comment (Bourkupine) [inline OBlock/src/enrolment.js:46]: Move this above the alert on line 45

**Review by carthy27 — APPROVED** [review]
lgtm frontend looks v good.

**Review by Bourkupine — APPROVED** [review]
Very happy with this. Unbelievable stuff. One minor nitpick but honestly doesn't matter too much.

Had a small suggestion for the front end:
image
Would be nice to be able to have a button here to quickly mark a student as complete/failed(or dropped out)

**Pull request merged**